tests/integration/jpa/appl/src/test/java/io/helidon/tests/integration/jpa/appl/test/ClientUtils.java

```diff
@@ -40,7 +47,30 @@ private ClientUtils() {
     public static void callTest(final String path) {
         WebTarget status = TARGET.path(path);
         Response response = status.request().get();
-        Validate.check(response.readEntity(String.class));
+        String responseStr = response.readEntity(String.class);
+        try {
+            Validate.check(responseStr);
+        } catch (JsonParsingException t) {
+            LOGGER.severe(() -> String.format("Response is not JSON: %s", t.getMessage()));
```

Comment:
This does not make any sense. Why is the `responseStr` not part of the sever log? Also not logging a stack trace is bad, as it is useless for troubleshooting.
Please merge these two log statements into one, and log the exception. 
(this is repeating, fix everywhere)

Reply:
This code is client side. It's evaluating incoming server response. If JSON parser fails, it must be really visible.

Reply:
No problem, just use this statement:
`LOGGER.log(Level.SEVERE, t, () -> String.format("Response is not JSON: %s, message: %s", t.getMessage(), responseStr));`
instead of the two lines you have there

Reply:
Done.